helpers/createHeaderParamsSnippet.js

```diff
@@ -6,10 +6,11 @@ const createHeaderParamsSnippet = (sortedParams) => {
   let headerSnippet = "";
 
   //Add cookie parameters
-  const cookieParams = getParametersByType(sortedParams, "cookie");
+  let cookieParams = getParametersByType(sortedParams, "cookie");
   if (cookieParams.length !== 0) {
     let safeParamName = toParamName(cookieParams[0].name);
     headerSnippet += `request.Headers.Add("cookie", $"${cookieParams[0].name}={${safeParamName}}`;
+    cookieParams = cookieParams.slice(1);
```

Comment:
Maybe check if `cookie.length >= 1` before the slice? probably fine and won't throw an error because it's javascript but good practice

Reply:
There is a check at line 10 to make sure there is at least one element in the array. If array is empty it skips this block

Reply:
but `cookieParams.slice(1)` drops the first element and only keeps the rest. There might not be more than 1 element

Reply:
It just returns an empty array, no error when only 1 element. I think it would error if there were 0 elements